tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -127,7 +130,7 @@ func (d *deployer) createCluster(zones []string, adminAccess string, yes bool) e
 		args = append(args, createArgs...)
 	}
 	args = appendIfUnset(args, "--admin-access", adminAccess)
-	args = appendIfUnset(args, "--master-count", "1")
+	args = appendIfUnset(args, "--master-count", fmt.Sprintf("%d", d.ControlPlaneSize))
```

Comment:
We are diverging from `--master-count` to `control-plane-size`. Why not keep it as is?

Reply:
Also, would be nice to make `--node-count` configurable also.